src/cookies.rs

```diff
@@ -1,7 +1,9 @@
 use std::collections::HashMap;
 
 const COOKIE_ATTRIBUTES: &str = "Path=/; SameSite=Lax; Secure; HttpOnly";
-const COOKIE_PREFIX: &str = "__Secure-";
+
+// Change to "__Secure-" for production, and "local-" for local development.
+const COOKIE_PREFIX: &str = "local-";
```

Comment:
I wonder if you can do something like you do with the redirect_uri (check fastly service version) so that you don't have to modify the code before deploying prod?

Reply:
Great suggestion, thank you!